sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/LogsHelper.cs

```diff
@@ -206,10 +206,6 @@ private static void ExtractProperties(ref string? message, IDictionary<string, s
                         {
```

Comment:
nit perf optimization: if message is null, then there is no need to check if each item.Key is "{OriginalFormat}".

Reply:
"{OriginalFormat}" is a part of `stateDictionary`. If we don't validate it will end-up in properties. Message check to null is a special case here.

Reply:
We need it! I meant to avoid the perf hit of checking every key against {OriginalFormat}. Instead check the key only if message is null, which is faster to check.

Reply:
something like if (message == null && item.key==originalformat) {
mesage = item.value.
}

Reply:
> something like if (message == null && item.key==originalformat) { mesage = item.value. }

It won't work for this scenario, originalformat will get added for the scenario when message is null.

Reply:
Got it. I missed the part that we *always* want to check for OriginalFormat in the keys to either skipping it or using it for message.